tsconfig.json

```diff
@@ -1,7 +1,6 @@
 {
   "exclude": [
     "node_modules",
-    "packages/blade-old/**",
```

Comment:
okay so the entire PR is just removing the `blade-old` package. we also need to modify a lot of config files for eg:
1. refactor the eslint files to just have ts things and remove everything else
2. what if we also remove the eslint, tsconfig, etc all the files  from inside the package/blade and keep them outside?
3. do we really need to keep the blade under `packages/blade` what else will we put in `packages`? examples and other stuff?

Reply:
> what if we also remove the eslint, tsconfig, etc all the files from inside the package/blade and keep them outside?

Don't think that would span out very well. eslint is already on the root. 

But if we put tsconfig outside then if you add new packages inside packages folder then they will also have all other moduleSuffixes stuff which is blade specific. 

> do we really need to keep the blade under packages/blade what else will we put in packages

Yeah? Even if we removed the blade-old, It's still a mono repo right. In future if we add other packages (figma plugins, examples, playground, docs) we should add them in packages folder.

Reply:
Lets keep it a mono-repo for adding playground, docs, etc 👍